pkgs/by-name/wc/wcurl/package.nix

```
@@ -0,0 +1,48 @@
+{
+  stdenv,
+  lib,
+  curl,
+  fetchFromGitLab,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wcurl";
+  version = "2024.07.10";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "debian";
+    repo = "wcurl";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-FYkG74uoXFNYT7tszDcdCPQCEG3ePOFBUgIUYpsAzb8=";
+  };
+
+  strictDeps = true;
+
+  dontBuild = true;
+
+  postPatch = ''
+    substituteInPlace wcurl \
+      --replace-fail '"curl "' '"${curl}/bin/curl "'
+  '';
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D wcurl -t $out/bin
```

```suggestion
    install -D wcurl -t $out/bin
    installManpage wcurl.1
```

(This needs `installShellFiles` in `nativeBuildInputs`)

---

Also maybe we can run tests in `checkPhase` since we do have shutil2

---

Thank you, calling the tests worked fine. Hopefully I put everything in the correct order - I made nix-hammer stop complaining about it, at least.